CommonControls/FileTypes/Animation/AnimationFile.cs

```diff
@@ -169,6 +169,17 @@ static AnimationHeader GetAnimationHeader(ByteChunk chunk)
             return header;
         }
 
+        public int GetIdFromBoneName(string name)
+        {
+            var result = Bones.Where(x => x.Name.Equals(name, StringComparison.OrdinalIgnoreCase));
+
+            var boneInfo = result.FirstOrDefault();
```

Comment:
Typically linq statements like this would be done in one line. Doing it the standard way makes the code faster to read
var boneInfo = Bones.Where.FirstOrDefault()